src/styles/index.scss

```diff
@@ -1,3 +1,112 @@
+$link-color: #00acdc;
+$text-regular-color: #616070;
+$text-bold-color: #060b35;
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
```

Comment:
We 99% of the time want `box-sizing: border-box` behaviour on every single element. Use `*` to define it.